docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -40,8 +41,27 @@ \subsection{Problem}
 
 \subsection{Inputs and Outputs}
 
-\wss{Characterize the problem in terms of ``high level'' inputs and outputs.  
-Use abstraction so that you can avoid details.}
+\subsubsection{Inputs}
+    \begin{itemize}
+        \item Personal consumer spending data
+        \item Location data
+        \item User Preferences
+        \item Historical Shopping Lists
+        \item User Reviews and Ratings
+        \item Shopping History
+    \end{itemize}
```

Comment:
I'm not gonna lie, I think you went a bit off script in my opinion. I think you can keep Personal Consumer Spending Data and Location Data however I think User Preferences and User Reviews and Ratings are a bit out of scope.

I think User Preferences could be a Stretch Goal but I don't think we're including User Reviews as part of the app.

I also think Personal Consumer Spending Data, Shopping History and Historical Shopping Lists are all the same way to repeat the same thing.

@Tanger71 @allanfang1 Any thoughts?

Reply:
Removed User Preferences and User Reviews and Ratings